schemas/test/en/test_placeholder_default_value.json

```diff
@@ -158,6 +158,121 @@
                                 ]
                             },
                             "type": "Interstitial"
+                        },
+                        {
+                            "type": "Question",
```

Comment:
I think it would be better suited to add test to `test_placeholder_transform`. This schema is just for the purposes of default values.

Reply:
Ok moved it to that schema, thanks.